src/Microsoft.Extensions.ProjectModel.Sources/IProjectContext.cs

```diff
@@ -18,6 +18,7 @@ public interface IProjectContext
         string Config { get; }
         string DepsJson { get; }
         string RuntimeConfigJson { get; }
+        string PackageLockFile { get; }
```

Comment:
If we wanted we could make this property smarter in the sense that it could validate that the file exists, if not returns `null`. Super minimal and I'm not even that convinced it's worth it. Food for thought unless you felt different.